Documentation/Blazorise.Docs/Pages/Docs/Extensions/DataGrid/Examples/DataGridAggregatesLargeDataExample.razor

```diff
@@ -53,7 +53,7 @@
             if ( e.ReadDataMode is DataGridReadDataMode.Virtualize )
                 response = employeeList.Skip( e.VirtualizeOffset ).Take( e.VirtualizeCount ).ToList();
             else if ( e.ReadDataMode is DataGridReadDataMode.Paging )
-                response = employeeList.Skip( ( e.Page - 1 ) * e.PageSize ).Take( e.PageSize ).ToList();
+                response = employeeList.Skip( (int)( e.Page - 1 ) * e.PageSize ).Take( e.PageSize ).ToList();
```

Comment:
As I explained in the pr description I don't consider this a final solution, it is here as temporary solution to be able to run the project. You sure we want this as a solution for the introduced breaking change?

Reply:
Yes, we want to do the things right in 2.0. This is going to a big update so there is no room around it. Change all occurrences of `int` to `long` where paging or total items is mentioned.

Regarding the `*Long` suffix proposal. That is not an option since it will just introduce additional APIs that we would have to maintain. We want to make it simpler.